code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/IdentityExtensionTests.java

```diff
@@ -471,6 +472,153 @@ public void test_handleHubSharedState_doesNotShareStateIfLegacyECIDDoesNotChange
 			.setXDMSharedEventState(sharedStateCaptor.capture(), eq(event), any(ExtensionErrorCallback.class));
 	}
 
+	// ========================================================================================
+	// handleUrlVariablesRequest
+	// ========================================================================================
+
+	@Test
+	public void test_handleUrlVariablesRequest_nullEvent_shouldNotThrow() {
+		// test
+		extension.handleUrlVariablesRequest(null);
+	}
+
+	@Test
+	public void test_handleUrlVariablesRequest_whenConfigAndEcidNotPresent_returnsNull() {
+		// setup
+		Event event = new Event.Builder(
+			"Test event",
+			IdentityConstants.EventType.EDGE_IDENTITY,
+			IdentityConstants.EventSource.REQUEST_IDENTITY
+		)
+			.build();
+		final ArgumentCaptor<Event> responseEventCaptor = ArgumentCaptor.forClass(Event.class);
+		final ArgumentCaptor<Event> requestEventCaptor = ArgumentCaptor.forClass(Event.class);
```

Comment:
For each of these tests, I don't see that you are using the requestEventCaptor. If not, can you simply use `any(Event.class)`?